src/server/acl/acl_commands_def.h

```diff
@@ -106,4 +107,56 @@ inline const std::vector<std::string> REVERSE_CATEGORY_INDEX_TABLE{
     "_RESERVED", "_RESERVED", "_RESERVED", "_RESERVED", "_RESERVED",  "_RESERVED",   "_RESERVED",
     "_RESERVED", "FT_SEARCH", "THROTTLE",  "JSON"};
 
+using BitfieldIndexPar =
+    std::pair<size_t /*index of family in the vector */, uint64_t /*bit index mask*/>;
+using CommandsIndexStore = absl::flat_hash_map<std::string, BitfieldIndexPar>;
+using RevCommandField = std::vector<std::string>;
+using RevCommandsIndexStore = std::vector<RevCommandField>;
+
+class CommandTableBuilder {
```

Comment:
Bread and butter of this PR. This allows us to build adhoc the index table for commands, since it's so large and I didn't even bother hardcoding the index structure.

Reply:
Ah you have a typedef one line above, matched the wrong line